tasks/gem.rake

```diff
@@ -20,7 +20,7 @@ namespace :gem do
 
     s.required_ruby_version = '>= 2.0'
 
-    s.add_runtime_dependency 'public_suffix', '~> 2.0', '>= 2.0.2'
+    s.add_runtime_dependency 'public_suffix', '>= 2.0.2', '< 4.0'
```

Comment:
Yeah, at some point I'll probably do a big code style audit. No worries for this PR.